gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/v4/impl/ApiNotificationServiceImpl.java

```diff
@@ -50,32 +51,38 @@ public ApiNotificationServiceImpl(
     }
 
     @Override
+    @Async
```

Comment:
should we have a dedicated executor for this @Async? 🤔

Reply:
I don't think we need a dedicated executor for this `@Async` at the moment. Currently, there is no custom executor defined for asynchronous tasks in our application, except for the one used with Lucene,  it might not be necessary